src/Protos/wallet.proto

```diff
@@ -84,7 +84,9 @@ message TransferRequest {
     repeated string hashed_attributes = 4; // the hashed attribute keys to transfer
 }
 
-message TransferResponse {} // not yet sure what to return
+message TransferResponse {
+    project_origin.common.v1.Uuid transfer_id = 1;
```

Comment:
Why this change?

Reply:
To make it on par with the new rest api, and claim api, where a ClaimId is also returns, it is so we later can query back on the task started to get a state of it if wanted.